src/Simulation/Common/SimulatorBase.cs

```diff
@@ -43,6 +43,7 @@ public abstract class SimulatorBase : AbstractFactory<AbstractCallable>, IOperat
 
         public abstract string Name { get; }
 
+        public bool EnableStackTracePrinting { get; set; } = true;
```

Comment:
I'd suggest adding an API documentation comment here. In particular, since this is only effective if `DisableExceptionPrinting()` has not been called, that would be good to note in a `<remark>` here to prevent confusion.

Reply:
Added comment.